packages/samples/angular/package.json

```diff
@@ -11,29 +11,30 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "16.0.2",
-    "@angular/common": "16.0.2",
-    "@angular/compiler": "16.0.2",
-    "@angular/core": "16.0.2",
-    "@angular/forms": "16.0.2",
-    "@angular/platform-browser": "16.0.2",
-    "@angular/platform-browser-dynamic": "16.0.2",
-    "@angular/router": "16.0.2",
+    "@angular/animations": "16.2.1",
+    "@angular/common": "16.2.1",
+    "@angular/compiler": "16.2.1",
+    "@angular/core": "16.2.1",
+    "@angular/forms": "16.2.1",
+    "@angular/platform-browser": "16.2.1",
+    "@angular/platform-browser-dynamic": "16.2.1",
+    "@angular/router": "16.2.1",
     "@coveo/atomic-angular": "2.14.2",
     "rxjs": "7.5.6",
     "tslib": "2.4.0",
     "zone.js": "0.13.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^16.0.2",
-    "@angular/cli": "16.0.2",
-    "@angular/compiler-cli": "16.0.2",
+    "@angular-devkit/build-angular": "^16.2.1",
+    "@angular/cli": "16.2.1",
+    "@angular/compiler-cli": "16.2.1",
```

Comment:
Bump, iirc because otherwise not TS 5.1 compat (for the peer deps resolution, does not seems to impact the output greatly)